### PR TITLE
[CBRD-24469] Unit tests for Java SP Server

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CBRD-24469

JUnit Task is added in the build script of ant (`build.xml`). The JUnit testing classes should be placed at the path, `com/cubrid/jsp/test`. The summary will be shown when you compile with `make && make jsp_unittest` in the binary path of Cmake.
```
...
test:
[ivy:retrieve] :: retrieving :: org.cubrid#cubrid-jsp
[ivy:retrieve]  confs: [test]
[ivy:retrieve]  0 artifacts copied, 6 already retrieved (0kB/3ms)
    [junit] Running com.cubrid.jsp.test.TestSample
    [junit] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.031 sec

dist:

BUILD SUCCESSFUL
...
```

If any JUnit test is failed, the `make jsp_unittest` command of CMake is going to fail as follow.
```
...
test:
[ivy:retrieve] :: retrieving :: org.cubrid#cubrid-jsp
[ivy:retrieve]  confs: [test]
[ivy:retrieve]  0 artifacts copied, 6 already retrieved (0kB/3ms)
    [javac] Compiling 1 source file to /home/cubrid/regex_hgryoo/build_x86_64_debug/jsp/bin/test-classes
    [javac] warning: [options] bootstrap class path not set in conjunction with -source 1.6
    [javac] 1 warning
    [junit] Running com.cubrid.jsp.test.TestSample
    [junit] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.027 sec

BUILD FAILED
/home/cubrid/regex_hgryoo/jsp/build.xml:86: Test com.cubrid.jsp.test.TestSample failed

Total time: 1 second
make[2]: *** [jsp/CMakeFiles/jsp_build.dir/build.make:83: jsp_build] Error 1
make[1]: *** [CMakeFiles/Makefile2:3087: jsp/CMakeFiles/jsp_build.dir/all] Error 2
make: *** [Makefile:172: all] Error 2
...
```

### CODE REVIEW

**Review by beyondykk9 — APPROVED** [review]

**Review by ctshim — APPROVED** [review]

**Review by shparkcubrid — APPROVED** [review]

File: jsp/build.xml [hunk 1]
```diff
@@ -52,29 +56,58 @@
...
             <fileset dir="${test.classes.dir}"/>
             <fileset dir="${classes.dir}"/>
             <fileset dir="${build.dir}"/>
             <fileset file="${dist.jar}"/>
         </delete>
     </target>
 
+    <target name="test-junit" depends="init">
+        <ivy:retrieve pattern="${dist.lib}/[artifact]-[revision].[ext]" conf="test"/>
+        <javac srcdir="${src.dir}" destdir="${test.classes.dir}" source="1.6" target="1.6" includeantruntime="no" encoding="UTF-8" classpathref="test.path">
```
Comment (hyunikn) [inline jsp/build.xml]: Should we update the source and target to 1.8 in this and line 65?

**Review by hyunikn — COMMENTED** [review]
I tried make jsp_unittest, and get the following error message: 
-=-=-=-=-=-=-=-=-=-=-=-=
BUILD FAILED
/home/cubrid/cubrid/jsp/build.xml:89: Problem: failed to create task or type junit
Cause: the class org.apache.tools.ant.taskdefs.optional.junit.JUnitTask was not found.
        This looks like one of Ant's optional components.
Action: Check that the appropriate optional JAR exists in
        -/usr/share/ant/lib
        -/home/cubrid/.ant/lib
        -a directory added on the command line with the -lib argument

File: jsp/build.xml (hunk 1 above)
Comment (hgryoo) [reply jsp/build.xml]: Ok, updating the version was omitted. thank you.

**Comment by hgryoo** [comment]
> I tried make jsp_unittest, and get the following error message: -=-=-=-=-=-=-=-=-=-=-=-= BUILD FAILED /home/cubrid/cubrid/jsp/build.xml:89: Problem: failed to create task or type junit Cause: the class org.apache.tools.ant.taskdefs.optional.junit.JUnitTask was not found. This looks like one of Ant's optional components. Action: Check that the appropriate optional JAR exists in -/usr/share/ant/lib -/home/cubrid/.ant/lib -a directory added on the command line with the -lib argument

Thank you. I've added ant-junit4 for the ant junit task. I think it is already installed in my environment. Could you please review it again?

**Review by youngjinj — APPROVED** [review]

**Comment by hgryoo** [comment]
> > I tried make jsp_unittest, and get the following error message: -=-=-=-=-=-=-=-=-=-=-=-= BUILD FAILED /home/cubrid/cubrid/jsp/build.xml:89: Problem: failed to create task or type junit Cause: the class org.apache.tools.ant.taskdefs.optional.junit.JUnitTask was not found. This looks like one of Ant's optional components. Action: Check that the appropriate optional JAR exists in -/usr/share/ant/lib -/home/cubrid/.ant/lib -a directory added on the command line with the -lib argument
> 
> Thank you. I've added ant-junit4 for the ant junit task. I think it is already installed in my environment. Could you please review it again?

I found `ant-junit` should be installed. Downloading ant-junit4 doesn't work. I'll merge it after I fix this problem.
```
sudo yum install ant-junit
```

**Review by hyunikn — APPROVED** [review]

**Pull request merged**